.travis.yml

```diff
@@ -21,16 +21,22 @@ env:
     - TEST_SUITE=integration INTEGRATION_INDEX=2
     - TEST_SUITE=integration INTEGRATION_INDEX=3
     - TEST_SUITE=static
+    - TEST_SUITE=js
 cache:
   apt: true
-  directories: $HOME/.composer/cache
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.nvm
 matrix:
   exclude:
     - php: 5.6.29
       env: TEST_SUITE=static
+    - php: 5.6.29
+      env: TEST_SUITE=js
 before_install: ./dev/travis/before_install.sh
 install: composer install --no-interaction --prefer-dist
 before_script: ./dev/travis/before_script.sh
 script:
   - test $TEST_SUITE = "static" && TEST_FILTER='--filter "Magento\\Test\\Php\\LiveCodeTest"' || true
-  - phpunit -c dev/tests/$TEST_SUITE $TEST_FILTER
+  - if [ $TEST_SUITE != "js" ]; then phpunit -c dev/tests/$TEST_SUITE $TEST_FILTER; fi
+  - if [ $TEST_SUITE == "js" ]; then grunt spec; fi
```

Comment:
Ok, I didn't use it as well, and didn't google for a solution just in case you know the answer.

It can wait until it become a problem "output is too big" some day (hopefully never).